helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -108,6 +108,9 @@ splunkObservability:
   # If you don't use AlwaysOn Profiling for Splunk APM, you can disable it.
   profilingEnabled: false
 
+  # HTTP timeout for sending data.
+  timeout: 20s
```

Comment:
I agree with Ryan. 20s for HTTP connection is too high. We shouldn't change it

Reply:
Thank you for your input. 
I rarely see the same context error in my k8s clusters that upload data to Splunk Observability, it probably only happens when backend or middle issues are present.
I'm okay with not adding a splunkObservability.timeout config, I thought it might be useful for Observability users.